src/components/notes/EditorCorner.spec.jsx

```diff
@@ -1,58 +1,56 @@
 import React from 'react'
-import { mount } from 'enzyme'
+import { render } from '@testing-library/react'
 
-jest.mock('components/notes/sharing', () => () => null)
-
-import { AppLike } from '../../../test/Applike'
-import SharingWidget from 'components/notes/sharing'
 import EditorCorner from './EditorCorner'
 
+jest.mock('components/notes/sharing', () => {
+  const MockSharing = () => <div data-testid="SharingWidget" />
+  return MockSharing
+})
```

Comment:
```
jest.mock('components/notes/sharing', () => () => (
  <div data-testid="SharingWidget"></div>
))
```
because it's more simple

Reply:
@trollepierre yea but then eslint complains about non-named component

Reply:
https://github.com/cozy/cozy-notes/pull/323
or https://github.com/cozy/cozy-notes/pull/321

Reply:
I approved https://github.com/cozy/cozy-notes/pull/323

I think we shouldn't use too many specific eslint configs and put everything in cozy-config

Reply:
#323 is merged, so we can make this change. :-)